Makefile

```diff
@@ -13,12 +13,13 @@ ARTIFACT := $(REGISTRY)$(IMAGE_PATH)/$(IMAGE_NAME)
 dep: ;
 
 lint:
-	docker run --rm -i -v "$(DIR):/mnt:ro" koalaman/shellcheck:v0.6.0 commands/*
+	docker run --rm -i -v "$(DIR):/mnt:ro" koalaman/shellcheck:v0.8.0 commands/*
 
 test:
 	docker build -t local/test/sdcli .
 	docker build -t local/test/sdclitests test
-	docker run -it local/test/sdclitests
+	docker run -i local/test/sdclitests
```

Comment:
I'm surprised to see `-i` here; is it needed?

Reply:
The pytest formatter seems to be happier with `-i`